Source/Core/Core/HW/Wiimote.cpp

```diff
@@ -53,12 +54,22 @@ void UpdateSource(unsigned int index)
   if (!ios)
     return;
 
-  const auto bluetooth = std::static_pointer_cast<IOS::HLE::BluetoothEmuDevice>(
-      ios->GetDeviceByName("/dev/usb/oh1/57e/305"));
-  if (!bluetooth)
-    return;
+  if (s_wiimote_sources[index] == WiimoteSource::Real)
+  {
+    const auto bluetooth = std::static_pointer_cast<IOS::HLE::BluetoothRealDevice>(
```

Comment:
This branch seems unnecessary in its current state...it's accessing the bluetooth and doing nothing with it.

Suggest fixing it (not sure what the correct solution would be) or just removing it if there's nothing to do:

```
if (s_wiimote_sources[index] == WiimoteSource::Emulated)
{
    const auto bluetooth = std::static_pointer_cast<IOS::HLE::BluetoothEmuDevice>(
        ios->GetDeviceByName("/dev/usb/oh1/57e/305"));
    if (!bluetooth)
      return;
bluetooth->AccessWiimoteByIndex(index)->SetSource(GetHIDWiimoteSource(index));
  }
```